.eslintrc.js

```diff
@@ -0,0 +1,103 @@
+// @flow
```

Comment:
Merged all eslint configs of the eslint repo along with those in the project in this config. I am open for suggestions if you think some of these are overkill or you are missing some more rules.